### PR TITLE
use detailed profiles in tests/nf-test.config

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: .github/workflows/gpu-tests.yml [hunk 1]
```diff
@@ -74,14 +74,14 @@ jobs:
...
     runs-on: "gpu"
-    name: "GPU | ${{ matrix.profile }} | ${{ matrix.shard }}"
+    name: "GPU Test | ${{ matrix.profile }} | ${{ matrix.shard }}"
     needs: nf-test-changes
     if: ${{ needs.nf-test-changes.outputs.modules != '[]' || needs.nf-test-changes.outputs.subworkflows != '[]' }}
     strategy:
       fail-fast: false
       matrix:
         shard: [1, 2]
-        profile: [docker_self_hosted, singularity] # conda?
```
Comment (mashehu) [inline .github/workflows/gpu-tests.yml:84]: Please add conda. We should exclude the modules that don't use it afterwards, like we do in the normal tests

File: tests/config/nextflow.config [hunk 2]
```diff
@@ -6,34 +6,75 @@ params {
...
-    memory = 3.GB
-    time = 2.h
+    resourceLimits = [
+        cpus: 2,
+        memory: '4.GB',
+        time: '1.h'
+    ]
 }
 
-if ("$PROFILE" == "singularity") {
```
Comment (mashehu) [inline tests/config/nextflow.config:14]: Will this still work for pytest?

Comment (sateeshperi) [reply tests/config/nextflow.config:14]: not sure...I can leave it in there but, I think it should work

Comment (sateeshperi) [reply tests/config/nextflow.config:14]: but, feel free to add changes and merge this

**Review by mashehu — APPROVED** [review]

**Pull request merged**